test/test_mps.py

```diff
@@ -8325,6 +8325,11 @@ class TestConsistency(TestCase):
         'rsqrt': ['b8', 'f32', 'i16', 'i32', 'u8'],
         'scatter': ['b8', 'f16', 'f32', 'i16', 'i32', 'i64', 'u8'],
         'scatter_add': ['b8', 'f16', 'f32', 'i16', 'i32', 'i64', 'u8'],
+        'scatter_reduce_amax': ['b8', 'f16', 'f32', 'i16', 'i32', 'i64', 'u8'],
+        'scatter_reduce_amin': ['b8', 'f16', 'f32', 'i16', 'i32', 'i64', 'u8'],
+        'scatter_reduce_mean': ['b8', 'f16', 'f32', 'i16', 'i32', 'i64', 'u8'],
```

Comment:
I don't think if we support reduce_mean. Only these are supported:
if(reduce == "sum" || reduce == "add")
            scatter_mode = MPSGraphScatterModeAdd;
          else if(reduce == "prod" || reduce == "multiply")
            scatter_mode = MPSGraphScatterModeMul;
          else if(reduce == "amax")
            scatter_mode = MPSGraphScatterModeMax;
          else if(reduce == "amin")
            scatter_mode = MPSGraphScatterModeMin;

Reply:
Good point. Oh - I have been running all of these tests with fallback and thus the reason for passing. All of these are actually calling into an unimplemented op (`scatter_reduce.two_out`).

Reply:
Updated